packages/sveltekit/test/config/withSentryViteConfig.test.ts

```diff
@@ -0,0 +1,90 @@
+import type { Plugin, UserConfig } from 'vite';
+
+import { withSentryViteConfig } from '../../src/config/withSentryViteConfig';
+
+describe('withSentryViteConfig', () => {
+  const originalConfig = {
+    plugins: [{ name: 'foo' }],
```

Comment:
m: Can we add a test case for when the `originalConfig` has no plugins?

Reply:
[0d84cee](https://github.com/getsentry/sentry-javascript/pull/7391/commits/0d84cee02a54f90cce0732349bb0661ecb07e3ba)